admin-part/src/main/kotlin/group/Summary.kt

```diff
@@ -90,7 +90,7 @@ internal fun AgentSummary.toDto() = SummaryDto(
     risks = riskCounts.total, //TODO remove after changes on frontend
     riskCounts = riskCounts,
     testDuration = testDuration,
-    tests = coverageByType.toTestTypeSummary(tests, durationByType),
+    tests = testTypeSummary(),
```

Comment:
Should we add "to" when a method converts an object to another object? toTestTypeSummary